dummysite-auth-module/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "dummysite-auth-module",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "dev": "vite",
+    "build": "run-p type-check build-only",
+    "preview": "vite preview",
+    "test:unit": "vitest",
+    "build-only": "vite build",
+    "type-check": "vue-tsc --noEmit -p tsconfig.vitest.json --composite false",
+    "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --fix --ignore-path .gitignore",
+    "format": "prettier --write src/"
+  },
+  "dependencies": {
+    "vue": "^3.2.47"
```

Comment:
Are we going to install [vuetify ](https://vuetifyjs.com/en/) here? I know this is the dummy website so maybe you want to go with a different component library or no componenet library to make it look different to the main site but if you do want to go with vuetify there is a quick way to scaffold a new site using `npm init vuetify@latest`